kani-driver/src/call_single_file.rs

```diff
@@ -107,8 +107,6 @@ impl KaniSession {
 
         if self.args.debug {
             flags.push("--log-level=debug".into());
-        } else if self.args.verbose {
-            flags.push("--log-level=info".into());
```

Comment:
Was this behavior undesirable? Not sure why this was changed.

The stderr logging was enough to fix, right?

Reply:
This was to fix #1631

Reply:
Hah, despite me being the one to file those issues, I'd forgotten what the verbose one was actually about.

Makes sense! :)

The CI failure is a mystery though...

Reply:
I found out the issue. Submitting a patch now.